vm/vm.go

```diff
@@ -74,6 +76,21 @@ func (im *Importer) Stats() string {
 	return im.s.String()
 }
 
+// AddExtraLabelsToImportPath - adds extra labels query params to given url path.
+func AddExtraLabelsToImportPath(path string, extraLabels []string) (string, error) {
```

Comment:
nit: would be great to have test for this func

Reply:
added test